### PR TITLE
Fix braces style

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Description
Fixes braces style for expansion functions.

## Feature(s) this PR does NOT handle:
Making a separate PR for vanilla functions.

## **Discord contact info**
AsparagusEduardo

### CODE REVIEW

File: src/battle_ai_switch_items.c [hunk 1]
```diff
@@ -2139,9 +2142,12 @@ static bool32 AI_OpponentCanFaintAiWithMod(u32 battler, u32 healAmount)
...
     u32 i;
     // Check special cases to NOT heal
-    for (i = 0; i < gBattlersCount; i++) {
-        if (GetBattlerSide(i) == B_SIDE_PLAYER) {
-            if (CanTargetFaintAiWithMod(i, battler, healAmount, 0)) {
+    for (i = 0; i < gBattlersCount; i++)
+    {
+        if (GetBattlerSide(i) == B_SIDE_PLAYER)
+        {
+            if (CanTargetFaintAiWithMod(i, battler, healAmount, 0))
```
[the remark this answers is not in the document]
Comment (AsparagusEduardo) [reply src/battle_ai_switch_items.c:2149]: Done

File: src/battle_util.c [hunk 2]
```diff
@@ -6621,14 +6622,18 @@ static u8 TryConsumeMirrorHerb(u32 battler, bool32 execute)
...
         gLastUsedItem = gBattleMons[battler].item;
         gBattleScripting.savedBattler = gBattlerAttacker;
         gBattleScripting.battler = gBattlerAttacker = battler;
         gProtectStructs[battler].eatMirrorHerb = 0;
-        if (execute) {
+        if (execute)
+        {
             BattleScriptExecute(BattleScript_MirrorHerbCopyStatChangeEnd2);
-        } else {
+        }
```
Comment (Bassoonian) [inline src/battle_util.c:6634]: More of a question than an actual review, but I believe it's inconsistent what happens to conditions (both if and else) when there's only one function that gets called if the condition is met. I've seen braces around it, I've also just seen a simple indentation. Probably something to sort out (possibly pret-side) and make more consistent, too

Comment (AsparagusEduardo) [reply src/battle_util.c:6634]: This is the logic I followed here:
- `else` has 2 lines, therefore braces are needed.
- If/else chain has at least 1 block with required braces, therefore all of them should have braces.

I could PR some of these style adjustments, as I've already PR'd a matching PR to pret for braces.
image 
image

Comment (AlexOn1ine) [reply src/battle_util.c:6634]: ~~Let's start a war in pret and move over the objectively correct style lol~~
```
if (condition) {
    // code
}
```

Comment (AsparagusEduardo) [reply src/battle_util.c:6634]: ~~pokeheartgold does it lol~~

**Review by DizzyEggg — DISMISSED** [review]

**Review by DizzyEggg — APPROVED** [review]

**Pull request merged**